etc/zfs/zfs-functions.in

```diff
@@ -104,7 +106,8 @@ fi
 
 # ----------------------------------------------------
 
-export ZFS ZED ZPOOL ZPOOL_CACHE ZFS_MOUNT ZFS_UNMOUNT ZFS_SHARE ZFS_UNSHARE
+export ZFS ZED ZPOOL ZPOOL_CACHE ZFS_LOAD_KEY ZFS_UNLOAD_KEY ZFS_MOUNT ZFS_UNMOUNT \
+	ZFS_SHARE ZFS_UNSHARE
```

Comment:
tab for continuation (should be 4 spaces)

Reply:
Why? Other lines use tab, too? Should we cleanup the whole file? (spaces an tabs are mixed)

Reply:
ZFS universally uses tab for indent and 4 spaces for continuation (see `zfs_daemon_start()` below in the file)

Reply:
Should I cleanup the whole file?

Reply:
The file is already fine, just change this to
```sh
export ZFS ZED ZPOOL ZPOOL_CACHE ZFS_LOAD_KEY ZFS_UNLOAD_KEY ZFS_MOUNT ZFS_UNMOUNT \
    ZFS_SHARE ZFS_UNSHARE
```